apps/condo/domains/organization/schema/ResetOrganizationService.js

```diff
@@ -113,6 +114,38 @@ const ResetOrganizationService = new GQLCustomSchema('ResetOrganizationService',
                     await OrganizationLink.softDelete(context, organizationLink.id, DV_SENDER)
                 }
 
+                const meters = await find('Meter', {
```

Comment:
you need to check. probably if you delete Meter you will trigger cascade cleaning for MeterReading.

If it so it's enough to delete `MeterResourceOwner`. And clean meters and old meter reading asynchronous in a future ...